vendor/jquery/component.dollar.js

```diff
@@ -0,0 +1,12 @@
+Ember.Component.reopen({
+  $(sel) {
+    Ember.assert(
```

Comment:
Sadly, because we are not currently transpiling this file, we get no "magic" with `Ember.assert`. This _roughly_ means that this will still run in production builds.

In general, I'd do something like [this](https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill/blob/276d3224535adaaac868435bf64a35a018c3f827/index.js#L59-L75) in the `index.js` to get normal transpilation support (_except for modules_):

```js
treeForVendor(rawVendorTree) {
    if (!this.shouldPolyfill) {
      return;
    }

    let babelAddon = this.addons.find(addon => addon.name === 'ember-cli-babel');

    let transpiledVendorTree = babelAddon.transpileTree(rawVendorTree, {
      babel: this.options.babel,

      'ember-cli-babel': {
        compileModules: false,
      },
    });

    return transpiledVendorTree;
  },
```

The `ember-cli-babel` addon specifically provides a `.transpileTree` for this purpose...

Reply:
done